src/components/CreateMenu.js

```diff
@@ -22,16 +26,64 @@ const propTypes = {
     // Callback to fire when a CreateMenu item is selected
     onItemSelected: PropTypes.func.isRequired,
 
+    // Menu items to be rendered on the list
+    menuOptions: PropTypes.arrayOf(
+        PropTypes.oneOf(Object.values(CONST.MENU_ITEM_KEYS)),
+    ).isRequired,
+
+    // Callback to fire when a AttachmentPicker item is selected
+    onAttachmentPickerSelected: PropTypes.func,
+
     ...windowDimensionsPropTypes,
 };
 
+const defaultProps = {
+    onAttachmentPickerSelected: () => {},
+};
+
 class CreateMenu extends PureComponent {
     constructor(props) {
         super(props);
 
         this.setOnModalHide = this.setOnModalHide.bind(this);
         this.resetOnModalHide = this.resetOnModalHide.bind(this);
         this.onModalHide = () => {};
+
+        const MENU_ITEMS = {
+            [CONST.MENU_ITEM_KEYS.NEW_CHAT]: {
+                icon: ChatBubble,
+                text: 'New Chat',
+                onSelected: () => redirect(ROUTES.NEW_CHAT),
+            },
+            [CONST.MENU_ITEM_KEYS.NEW_GROUP]: {
+                icon: Users,
+                text: 'New Group',
+                onSelected: () => redirect(ROUTES.NEW_GROUP),
+            },
+            [CONST.MENU_ITEM_KEYS.REQUEST_MONEY]: {
+                icon: MoneyCircle,
+                text: 'Request Money',
+                onSelected: () => redirect(ROUTES.NEW_CHAT),
+            },
+            [CONST.MENU_ITEM_KEYS.SPLIT_BILL]: {
+                icon: Receipt,
+                text: 'Split Bill',
+                onSelected: () => redirect(ROUTES.NEW_CHAT),
+            },
+            [CONST.MENU_ITEM_KEYS.ATTACHMENT_PICKER]: {
+                icon: Paperclip,
+                text: 'Add Attachment',
+                onSelected: () => this.props.onAttachmentPickerSelected(),
```

Comment:
It would be good to keep `MENU_ITEMS` out of the class definition, but I'm guessing we can't do that due to the 'this' reference in `this.props.onAttachmentPickerSelected`, right?

Reply:
yup, exactly. I originally had it defined outside but had to pull it in for the attachment logic